kiwixlib/build.gradle

```diff
@@ -19,9 +19,9 @@ android {
 }
 
 dependencies {
-  compile fileTree(dir: 'libs', include: ['*.jar'])
-  androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
```

Comment:
Was changing these lines a part of this issue ?

Reply:
When we are upgrading the **compileSdkVersion** and the **targetSdkVersion**, we must also upgrade to use the latest Android plugin version which requires a minimum Gradle version. So after upgrading, we must use the latest dependency configuration. And changing one thing and leaving the other often leads to crashes. So changing these lines becomes a part and parcel of this issue and I have tested the app everything is up and running. There's no harm in changing these lines. The PR can be merged safely. :)

Reply:
Just a side note kiwixlib is separated from the main project and while this code is likely compiled by a sync. It won't be run unless using kiwix-build.

Reply:
@mhutti1 I do not think these changes were necessary as you stated [here](https://github.com/kiwix/kiwix-android/issues/376#issuecomment-365532141). The old configurations would still do the part to the best of my knowledge as they both are 2 separate modules. It's your call.